src/main/java/io/strimzi/kafka/quotas/StaticQuotaCallback.java

```diff
@@ -110,6 +115,25 @@ public void configure(Map<String, ?> configs) {
     private class StorageChecker implements Runnable {
         private final Thread storageCheckerThread = new Thread(this, "storage-quota-checker");
         private volatile boolean running = false;
+        private final Metric kafkaBrokerAllUsedBytes = Metrics.newGauge(mNameCreate("AllUsedBytes"), new Gauge<Long>() {
+            public Long value() {
+                return storageUsed.get();
+            }
+        });
+        private final Metric getKafkaBrokerSoftLimitBytes = Metrics.newGauge(mNameCreate("SoftLimitBytes"), new Gauge<Long>() {
+            public Long value() {
+                return storageQuotaSoft;
+            }
+        });
+
+        private MetricName mNameCreate(String name) {
```

Comment:
The `m` is a bit cyptic. What's wrong with `metricName`?

Reply:
I can change this no problem!